doc/classes/Callable.xml

```diff
@@ -160,7 +188,11 @@
 		<method name="is_custom" qualifiers="const">
 			<return type="bool" />
 			<description>
-				Returns [code]true[/code] if this [Callable] is a custom callable. Custom callables are created from [method bind] or [method unbind]. In GDScript, lambda functions are also custom callables.
+				Returns [code]true[/code] if this [Callable] is a custom callable. Custom callables are used:
+				- for binding/unbinding arguments (see [method bind] and [method unbind]);
+				- for representing methods of built-in [Variant] types (see [method create]);
+				- for representing lambda functions in GDScript;
+				- and other.
```

Comment:
"And other" strikes me a little odd. What could "other" possibly entail? What's the engine hiding from us?

Reply:
```
16 matches
core/extension/gdextension_interface.cpp: 1
 47: 31: class CallableCustomExtension : public CallableCustom {
core/object/callable_method_pointer.h: 6
 40: 39: class CallableCustomMethodPointerBase : public CallableCustom {
 81: 35: class CallableCustomMethodPointer : public CallableCustomMethodPointerBase {
127: 38: class CallableCustomMethodPointerRet : public CallableCustomMethodPointerBase {
174: 39: class CallableCustomMethodPointerRetC : public CallableCustomMethodPointerBase {
227: 41: class CallableCustomStaticMethodPointer : public CallableCustomMethodPointerBase {
268: 44: class CallableCustomStaticMethodPointerRet : public CallableCustomMethodPointerBase {
core/variant/callable_bind.h: 2
 37: 26: class CallableCustomBind : public CallableCustom {
 65: 28: class CallableCustomUnbind : public CallableCustom {
core/variant/variant_callable.h: 1
 37: 23: class VariantCallable : public CallableCustom {
modules/gdscript/gdscript_lambda_callable.h: 2
 44: 30: class GDScriptLambdaCallable : public CallableCustom {
 71: 34: class GDScriptLambdaSelfCallable : public CallableCustom {
modules/gdscript/gdscript_rpc_callable.h: 1
 39: 27: class GDScriptRPCCallable : public CallableCustom {
modules/gdscript/gdscript_utility_callable.h: 1
 38: 31: class GDScriptUtilityCallable : public CallableCustom {
modules/mono/managed_callable.h: 1
 40: 23: class ManagedCallable : public CallableCustom {
modules/mono/signal_awaiter_utils.h: 1
 41: 26: class BaseSignalCallable : public CallableCustom {
```

Reply:
One could then say custom Callables may be generated and used internally, perhaps?